lib/Elastica/Multi/ResultSet.php

```diff
@@ -39,44 +37,13 @@ class ResultSet implements \Iterator, \ArrayAccess, \Countable
     /**
      * Constructs ResultSet object.
      *
-     * @param \Elastica\Response       $response
-     * @param array|\Elastica\Search[] $searches
+     * @param \Elastica\Response $response
+     * @param BaseResultSet[]
```

Comment:
It is aliased in the import, because there is already a ResultSet in the Multi namespace.